src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1262,7 +1262,17 @@ PUBLIC_API STATUS peerConnectionOnDataChannel(PRtcPeerConnection, UINT64, RtcOnD
 PUBLIC_API STATUS peerConnectionOnConnectionStateChange(PRtcPeerConnection, UINT64, RtcOnConnectionStateChange);
 
 /**
- * Load the sdp field of PRtcSessionDescriptionInit with latest local session description
+ * Load the sdp field of PRtcSessionDescriptionInit with pending or current local session description
+ *
+ * @param[in] PRtcPeerConnection Initialized RtcPeerConnection
+ * @param[in,out] PRtcSessionDescriptionInit IN/PRtcSessionDescriptionInit whose sdp field will be modified.
+ *
+ * @return STATUS code of the execution. STATUS_SUCCESS on success
+ */
+PUBLIC_API STATUS peerConnectionGetLocalDescription(PRtcPeerConnection, PRtcSessionDescriptionInit);
```

Comment:
Why are we introducing a new API here instead of modifying the existing and adding an ENUM?

Reply:
As I mentioned in #386 and the beginning of this PR, it was from W3C WebRTC API which has 3 methods to retrieve local descriptions:

* currentLocalDescription() - kvs supported
* pendingLocalDescription()
* localDescription()

When you need to acquire local SDP that is not offered to the remote yet, you will need to use pendingLocalDescription(), or localDescription(), not currentLocalDescription().

There isn't a good document describes what they are but [this one](https://docs.w3cub.com/dom/webrtc_api/connectivity/#Pending_and_current_descriptions) seems to be the best one.

I believe @Sean-Der knows about this much better than I.